libcloudforensics/providers/gcp/internal/compute.py

```diff
@@ -1224,6 +1226,112 @@ def DetachServiceAccount(self) -> None:
       raise errors.ServiceAccountRemovalError('Service account detatchment '
           'failure: {0:s}'.format(str(exception)), __name__)
 
+  def _NormaliseFirewallL4Config(self, l4config: List[Any]) -> List[Any]:
+    """Normalise l4config dict key names that differ between policies and
+    firewalls.
+
+    Args:
+      l4config List[Any]: the l4config to be normalised
+
+    Returns:
+      List[Any]: the normalised l4config"""
+    normalised_l4config = []
+    for config in l4config:
+      normalised = {}
+      if 'ipProtocol' in config:
+        normalised['ip_protocol'] = config['ipProtocol']
+      else:
+        normalised['ip_protocol'] = config['IPProtocol']
```

Comment:
If `ipProtocol` is not in `config`, is it always guaranteed that `IPProtocol` is?

Reply:
Yeah according to the documentation each "L4config" will have either an `ipProtocol` if it is a firewall policy or `IPProtocol` if it is a standard firewall rule. Not sure why they are named differently :smiley: . `elif` is probably more suitable here though will change.